Src/BootCamp.Chapter/Checks.cs

```diff
@@ -9,31 +7,192 @@ public static class Checks
+                    personCount = 1;
+                }
+            }
+
+            return MakeStuffWithInput.MakefullstringRichestPeople(personCount, richestName, amount);
+        }
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            string biggestLossName = "";
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var a = new Account(peopleAndBalances[i]);
+                if (a.MoreThan1Balance())
+                {
+
+                }
+                else if (biggestLossName == "")
+                {
+                    biggestLossName = a.GetName();
+                    amount = a.BiggestLoss();
+                }
+                else if (amount == a.BiggestLoss())
+                {
+                    if (personCount == 1)
+                    {
+                        biggestLossName += " and " + a.GetName();
+                        personCount++;
+                    }
+                    else
+                    {
+                        biggestLossName = biggestLossName.Replace(" and ", ", ");
+                        biggestLossName += " and " + a.GetName();
+                        personCount++;
+                    }
+                }
+                else if (amount > a.BiggestLoss())
+                {
+                    biggestLossName = a.GetName();
+                    amount = a.BiggestLoss();
+                    personCount = 1;
+                }
+            }
+
+            return MakeStuffWithInput.MakefullstringBiggestLossPeople(biggestLossName, amount);
         }
 
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            string richestName = "";
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                var a = new Account(peopleAndBalances[i]);
+                if (i == 0)
+                {
+                    richestName = a.GetName();
+                    amount = a.HighestBalanceEver();
+                }
+                else if (amount == a.HighestBalanceEver())
+                {
+                    if (personCount == 1)
+                    {
+                        richestName += " and " + a.GetName();
+                        personCount++;
+                    }
+                    else
+                    {
+                        richestName = richestName.Replace(" and ", ", ");
+                        richestName += " and " + a.GetName();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)